README.md

```diff
@@ -44,27 +44,23 @@ end
 #### Services config
 The services config is YAML file containing a list (array) of services.
 Each item in the services list contains:
-- __base_uri__: one or more host:port combinations for each environment of the API.
+- __base_uri__: one or more host:port combinations for each environment of the API. NOTE: Including the port is important. The host+port have to match exactly and the system will _not_ infer anything. So for example, betamocks will not assume that an `https` endpoint will use port 443. You must specify it. 
```

Comment:
Wordy.  Also, I'd prefer a code change to forcefully require `port` as opposed to strongly worded documentation telling them, "or else :angry: "